ui/apps/platform/cypress/helpers/inviteUsers.js

```diff
@@ -0,0 +1,10 @@
+import { accessModalSelectors } from '../integration/accessControl/accessControl.selectors';
+
+export function checkInviteUsersModal() {
+    // check that the modal opened
+    cy.get(`${accessModalSelectors.title}:contains("Invite users")`);
+    cy.get(`${accessModalSelectors.button}:contains("Invite users")`);
+
+    // test closing the modal
+    cy.get(`${accessModalSelectors.button}:contains("Cancel")`).click();
```

Comment:
Good catch.

I'd like to make this addition in the next PR, because it's stacked behind this, I already added more tests, and I'd like to avoid rebase-heck.